ethvm/main.cpp

```diff
@@ -83,10 +84,11 @@ int main(int argc, char** argv)
 	Address sender = Address(69);
 	Address origin = Address(69);
 	u256 value = 0;
-	u256 gas = state.gasLimitRemaining();
+	u256 gas = 0;
```

Comment:
Alter `State` to `Block` and use the `state()` method where a `State` object is called for later.